Makefile

```diff
@@ -222,10 +227,12 @@ coverage: ##@test Run tests once in NodeJS generating coverage
 # Other
 #--------------
 react-native-desktop: export TARGET_OS ?= $(HOST_OS)
+react-native-desktop: export _NIX_PURE ?= true
```

Comment:
The macOS build isn't currently pure, right?

Reply:
hmm, this is just for react-native, so I guess it's fine.

Reply:
Yep.